addons/mrp/models/mrp_production.py

```diff
@@ -74,7 +74,7 @@ def _get_default_is_locked(self):
     name = fields.Char(
         'Reference', copy=False, readonly=True, default=lambda x: _('New'))
     priority = fields.Selection(
-        PROCUREMENT_PRIORITIES, string='Priority', default='0', index=True,
```

Comment:
Isn't this used by the _order clause in list view? `_order = 'priority desc, date_planned_start asc,id'`

Reply:
If it was only `_order = 'priority desc', sure. But with the other fields in the order by, postgresql won't use the index (during my test with a lot of populated data). It seems logical to me because postgresql needs to order also by `date_planned_start` and `id`, and I don't think he will used 3 indexes to recreate the order instead of simple scan (maybe I am wrong for a big database, more than 100K of MO). I maybe need to test with this kind of scale ?
Maybe we should add a multiple index to reflect the order_by (also for picking and maybe stock_move), but it is a other topic, and I should try with the populate tool.